app/src/main/java/com/android/unio/ui/authentication/Welcome.kt

```diff
@@ -209,16 +215,7 @@ fun handleAuthentication(email: String, password: String, context: Context) {
         ToastUtils.showToast(context, context.getString(R.string.welcome_toast_sign_in))
 
         if (signInResult.user?.isEmailVerified == false) {
-          signInResult.user.sendEmailVerification().addOnCompleteListener {
-            if (it.isSuccessful) {
-              ToastUtils.showToast(
-                  context, context.getString(R.string.welcome_toast_verification_sent))
-            } else {
-              ToastUtils.showToast(
-                  context, context.getString(R.string.welcome_toast_verification_sent_failed))
-              Log.e("WelcomeScreen", "Failed to send verification email", it.exception)
-            }
-          }
```

Comment:
Fair deletion 👍 Since we now redirect the user to the `Auth` screen if the user closes the app when they have to verify their email, this edge case disappears.